tests/e2e/test_e2e.py

```diff
@@ -178,12 +178,12 @@ def test_argument_file_without_path(self):
             config.parse_args(["--argumentfile"])
         assert 'Argument file "" does not exist' in str(err)
 
-    def test_use_nested_argument_file(self, test_data_dir):
+    def test_use_circular_argument_file(self, test_data_dir):
```

Comment:
These changes are in the scope of another PR #774 and should be removed from here.

Reply:
Rebased after merging #774